src/App.js

```diff
@@ -10,7 +10,7 @@ class App extends Component {
           rel="stylesheet"
           href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
           integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
-          crossorigin="anonymous"
+          crossOrigin="anonymous"
```

Comment:
is this intentional?

Reply:
According to MDN this was correct: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link